study.md

```diff
@@ -33,7 +38,10 @@ How does Ember use the URL to load view-state? Which layers in Ember are
 responsible for which tasks?
 
 ```md
-<!-- your answer here -->
+Ember uses the url to load a view-state when the router gets hit then goes to that routes file and loads the model then the template will render.
+The router is responsible for parsing the url to see if a route exists.
+If the route does exist ember then will look for the file in the routes folder that matches the parsed url and then will exescute the model in that route.
+When the data comes from the model it will then be renderd in the template.
```

Comment:
Yes, the router is the layer that gets a URL string and uses it to load a particular page.  The router talks to the appropriate model, the model talks to the controller and template to render the contents on the page.